x/observer/types/message_vote_block_header.go

```diff
@@ -59,10 +58,6 @@ func (msg *MsgVoteBlockHeader) ValidateBasic() error {
 		return cosmoserrors.Wrapf(sdkerrors.ErrInvalidAddress, err.Error())
 	}
 
-	if !chains.IsHeaderSupportedChain(msg.ChainId) {
```

Comment:
The `IsHeaderSupportedChain` is not used atm. Should remove or keep?

Reply:
Removed it with SupportMerkleProof, as it is unclear once we need back